src/sparseml/transformers/utils/initializers.py

```diff
@@ -0,0 +1,203 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Functionality for initializing a transformer model from a given path
+"""
+
+import logging
+import math
+import os
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Any, Optional, Union
+
+from transformers import AutoConfig, AutoModel, AutoTokenizer, TrainingArguments
+
+from sparseml.optim import parse_recipe_variables
+from sparseml.transformers.sparsification import Trainer
+from sparseml.transformers.utils.helpers import TaskNames
+from sparseml.transformers.utils.load_task_model import load_task_model
+
+
+__all__ = [
+    "initialize_model",
+    "initialize_tokenizer",
+    "initialize_trainer",
+    "initialize_config",
+    "resolve_sequence_length",
+]
+
+_LOGGER = logging.getLogger(__name__)
+
+
+# TODO: Let's not take this for granted. We should be able to
+# device argument
+@dataclass
+class ForceCPUTrainingArguments(TrainingArguments):
```

Comment:
what is the expected usage?

Reply:
This is a legacy object from the old export. Now when I think about this, it should go.